domains/barath.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "BARATH-XD",
+        "email": "crazysbots@gmail.com"
+    },
+    "record": {
+        "URL": "barath.is-a.dev"
```

Comment:
URL needs protocol, also you cant point the is-a.dev domain to the one you are trying to get